preview-src/docs-sub-hub.adoc

```diff
@@ -0,0 +1,96 @@
+= What is Neo4j?
+:page-layout: docs-ndl
+:page-theme: docs
+:page-role: explainer
+:page-show-home-link: true
+:page-hide-nav-title: true
+:page-disablefeedback: true
+:page-toclevels: 1
+
+
+ifndef::backend-pdf[]
+++++
+<div class="video widget responsive-embed">
+<iframe width="680" height="425" src="https://www.youtube.com/embed/urO5FyP9PoI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+++++
+endif::[]
+
+
+
+[.cards.selectable]
+== Ecosystem
+
+In imperdiet ex eget mi fermentum, sed egestas tortor interdum.
+Aliquam vehicula nisl non pharetra suscipit.
+Nulla tortor sapien, eleifend non magna nec, auctor cursus velit.
+Phasellus in purus massa.
+Sed maximus tortor et viverra lacinia.
+Nullam fermentum purus at odio rutrum semper.
+Nunc lacinia, ipsum sed pharetra luctus, orci magna condimentum enim, sit amet condimentum nunc sem at lorem.
+
+
+=== Cypher Query Language
+
+[.icon]
+image:icons/ndl/command-line.svg[]
+
+[.description]
+Phasellus in purus massa.
+Sed maximus tortor et viverra lacinia.
+Nullam fermentum purus at odio rutrum semper.
+
+[.link]
+link:{docs-home}/cypher-manual[]
+
+
+=== Drivers and connectors
+
+[.icon]
+image:icons/ndl/command-line.svg[]
+
+[.description]
+Phasellus in purus massa.
+Sed maximus tortor et viverra lacinia.
+Nullam fermentum purus at odio rutrum semper.
+
+[.link]
+link:{docs-home}/connectors[]
+
+
+=== Neo4j Tools
+
+[.icon]
+image:icons/ndl/command-line.svg[]
+
+[.description]
+Phasellus in purus massa.
+Sed maximus tortor et viverra lacinia.
+Nullam fermentum purus at odio rutrum semper.
+
+[.link]
+link:{docs-home}/connectors[]
+
+
+[.next-steps]
+== Start exploring
+
+Lorem ipsum dolor sit est.
+
+
+. *Tutorial: Getting started with Cypher*
++ 
+Learn stuff
+. *Working with data sets*
++
+Learn stuff
+. *Step 3 goes here*
```

Comment:
Question: with these changes, now every time we use `.` it will turn into the blue circled number?

Reply:
No, it's using a role that is applied to the whole list, in this case `[.next-steps]`, so only lists with that role would have the styles applied.

Also, the css styles for this role are only applied when the `page-layout` attribute is set to `docs-ndl`, which we will use for all the new index pages. If we want to use this style of list anywhere else we could remove that requirement.

Reply:
True, I missed the `[.next-steps]` line. I was wondering if that could be helpful for tutorial pages at some point, but we can revisit that later. Thanks for explaining!

Reply:
Yes, this could become a reusable pattern can apply to existing docs - tutorials are a good example

Reply:
We are missing the feedback widget. I guess we can use the same style, just update with NDL colors?

Reply:
I've restored the feedback widget - it disabled it at some point in the pages while developing

Reply:
I've added a `border-radius` to display slightly rounded corners